Makefile

```diff
@@ -1,5 +1,3 @@
-export AFL ?= $(HOME)/.afls/afl-1.83b
```

Comment:
It took me a few mins to figure out what was going on here and why my commands were failing, I figured it was less surprising to require setting it.